workshops/hangman/README.md

```diff
@@ -0,0 +1,284 @@
+
+```java
+public void hang() {
+  numIncorrect++;
+  switch(numIncorrect){
+	case 1:
+	  body[1] = 'O';
+	  break;
+	case 2:
+	  body[5] = '|';
+	  break;
+	case 3:
+	  body[4] = '\\';
+	  break;
+	case 4:
+	  body[6] = '/';
+	  break;
+	case 5:
+	  body[8]='/';
+	  break;
+	case 6:
+	  body[10] = '\\';
+	  break; 
+  }
+}
+```
+
+If we just look at the ‘case 1:’ segment of the code, our code is basically saying that it will set the second element of body (Arrays are 0 indexed) to ‘O’. The illustrations below indicate which index of the body Arrays corresponds to which body part of the man. Also, you may have noticed that for the left arm and right leg, we used 2 backward slashes, not 1. This is because ‘\’ alone is a special Java character, like \n for linebreak, and Java will not recognize it. So we have to add an extra ‘\’ so that it prints out an actual backslash.
+
+```
++---+---+----+----+   +---+---+---+----+
+| 0 | 1 | 2  | 3  |   |   | O |   | \n |
++---+---+----+----+   +---+---+---+----+
+| 4 | 5 | 6  | 7  |   | \ | | | / | \n |
++---+---+----+----+   +---+---+---+----+
+| 8 | 9 | 10 | 11 |   | / |   | \ | \n |
++---+---+----+----+   +---+---+---+----+
+```
+
+## Part 3: Trying out the Man Class
+
+Believe or not, we’re done with the Man class! Before moving onto building the hangman game, let’s test out the hangman first and see if it is printed out correctly. All the test code will be in the main() method because all Java code is run in the main() method. In order to test the man, we want to create a new Man object which we can call ‘m’, hang it, then print it until it is completely hanged. We can test this using a for loop. However, there is a catch.
+Repl is set up to run our Main class, so we will need to run a few commands to try out our Man class now. Java is a compiled language, meaning that first our program must be converted to machine code before being run. We can do this by running the java compiler.
+First type javac Man.java into the console on the right of your screen to compile our Man class. This will generate a `Man.class` file (it might not show up in the repl file explorer, but don’t worry - it should be there!). Next type java Man and see what happens. You should see all the different stages of the man being hanged before dying.
+
+```java
+public static void main(String[] args) {
```

Comment:
this step is giving me:

```
error: class found on application class path: Man
```

Reply:
That usually happens when you run `java Man.java` instead of `java Man`. Could that be the reason?